django/core/management/commands/inspectdb.py

```diff
@@ -8,6 +8,15 @@
 from django.db import DEFAULT_DB_ALIAS, connections
 
 
+class DO_NOTHING():
```

Comment:
the introspect command will join the `extra_params` key (ie: `on_delete`) with the `__repr__` of its value. See L148 (was L135).

`repr("models.CASCADE")` will be literally be `"models.CASCADE"`, resulting in `on_delete="models.CASCADE"`, where it should be `on_delete=models.CASCADE`